quesma/model/typical_queries/hits.go

```diff
@@ -29,6 +29,7 @@ type Hits struct {
 	addScore       bool // true <=> we add hit.Score field to the response (whose value is always 1)
 	addVersion     bool // true <=> we add hit.Version field to the response (whose value is always 1)
 	indexName      string
+	useDefaultID   bool
```

Comment:
I think it now makes sense to place a little comment explaining what the default ID is in this context

Reply:
It relies on the presence of a timestamp field now.